src/box/space_upgrade.h

```diff
@@ -60,15 +61,17 @@ space_upgrade_def_delete(struct space_upgrade_def *def)
 }
 
 /**
- * Creates a space upgrade state from a definition and the new space format.
- * Returns NULL and sets diag on error.
+ * Creates a space upgrade state from a definition, space name, primary key
+ * definition and the new space format. Returns NULL and sets diag on error.
  * The reference count of the new state is set to 1.
  */
 static inline struct space_upgrade *
-space_upgrade_new(const struct space_upgrade_def *def,
-		  struct tuple_format *format)
+space_upgrade_new(const struct space_upgrade_def *def, const char *name,
+		  const struct key_def *pk_def, struct tuple_format *format)
```

Comment:
Let's also pass `space_id`? We could use it for consistent error messages:

```C
say_info("space upgrade started, space '%s' (%u), function '%s' (%u)",
```

([link](https://github.com/tarantool/tarantool-ee/blob/6a93559749dc4a8831160202d3963c614207e41e/src/box/space_upgrade.c#L609))

Reply:
We could use space lookup by name to fetch space or vice versa - by space id. So I guess we should leave either space id or space name.

Reply:
We can't lookup the space by name, because, strictly speaking, the space may not exist when we log the message. Agreed to remove ids from the all log messages instead.

Reply:
Ok, renamed param `name` to `space_name`.

Reply:
Rebased to master also.